app/controllers/rentals_controller.rb

```diff
@@ -0,0 +1,65 @@
+require 'date'
+
+class RentalsController < ApplicationController
+
+  def create
+    if params[:movie_id].nil? || params[:customer_id].nil? || (Movie.find_by(id: params[:movie_id])).nil? || (Customer.find_by(id: params[:customer_id])).nil?
+      render json: {ok: false}, status: :not_found
+    else
+      movie = Movie.rentable_movie?(params[:movie_id])
+      customer = Customer.find_by(id: params[:customer_id])
+      today = Date.today
+
+      rental_data = {
+        movie_id: (params[:movie_id]),
+        customer_id: (params[:customer_id]),
+        check_out_date: today.to_s,
+        due_date: (today + 7).to_s
+      }
+
+      if movie
+        rental = Rental.new(rental_data)
+      end
+
+      if movie && rental.save
```

Comment:
you just checked for `movie`'s truthiness a line above, maybe there's a way to refactor this logic?